Source/WebCore/accessibility/AccessibilityNodeObject.h

```diff
@@ -189,6 +192,11 @@ class AccessibilityNodeObject : public AccessibilityObject {
     bool m_isARIAGridCell { false };
     // Only used by AccessibilitySVGObject, but placed here to use space that would otherwise be taken by padding.
     bool m_isSVGRoot { false };
+
+    // Only used by objects representing labels to use space that would otherwise be taken by padding.
```

Comment:
Hmm, this sentence may not read clearly? "Only used by objects representing labels to use space"

If we take my other suggestion, we could say something like:

`// Only used by isNativeLabel() objects. Placed here to use space that would otherwise be taken by padding.`